telethon/client/telegrambaseclient.py

```diff
@@ -890,7 +892,8 @@ async def _clean_exported_senders(self: 'TelegramClient'):
     async def _get_cdn_client(self: 'TelegramClient', cdn_redirect):
         """Similar to ._borrow_exported_client, but for CDNs"""
         # TODO Implement
-        raise NotImplementedError
+        # raise NotImplementedError
+        from .telegramclient import TelegramClient
```

Comment:
This seems quite hacky, but eh, as long as it works, I don't really care much in v1.